src/lib/models/ConfigModel.ts

```diff
@@ -147,7 +163,7 @@ abstract class YamlFormatter {
 
 	private getDefaultRuleFor(engineName: string, ruleName: string): Rule|null {
 		try {
-			return this.defaultContext.rules.getRule(engineName, ruleName);
+			return this.allDefaultRules.getRule(engineName, ruleName);
```

Comment:
This is the only place allDefaultRules is used. So like I said, I could have just passed in the getDefaultRuleFor method instead... but this was already here - so just keeping things as is.